components/binding-usage-controller/docs/usage-kind.md

```diff
@@ -0,0 +1,33 @@
+# UsageKind
+## Overview
```

Comment:
Since it's not a README, it doesn't have to follow the template. I'd get rid of the Overview heading :)